code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -83,6 +83,42 @@
 		M.satiety += 30
 	. = ..()
 
+/datum/reagent/consumable/cooking_oil
+	name = "Cooking Oil"
+	id = "cooking_oil"
+	description = "A variety of cooking oil derived from fat or plants. Used in food preparation and frying."
+	color = "#EADD6B" //RGB: 234, 221, 107 (based off of canola oil)
+	taste_mult = 0.8
+	taste_description = "oil"
+	nutriment_factor = 7 * REAGENTS_METABOLISM //Not very healthy on its own
+	metabolization_rate = 10 * REAGENTS_METABOLISM
+	var/fry_temperature = 450 //Around ~350 F (117 C) which deep fryers operate around in the real world
+	var/boiling //Used in mob life to determine if the oil kills, and only on touch application
+
+/datum/reagent/consumable/cooking_oil/reaction_obj(obj/O, reac_volume)
+	if(holder && holder.chem_temp >= fry_temperature)
+		if(isitem(O))
+			O.loc.visible_message("<span class='warning'>[O] rapidly fries as it's splashed with hot oil! Somehow.</span>")
+			var/obj/item/reagent_containers/food/snacks/deepfryholder/F = new(get_turf(O))
+			F.fry(O, volume)
+
+/datum/reagent/consumable/cooking_oil/reaction_mob(mob/living/M, method = TOUCH, reac_volume, show_message = 1, touch_protection = 0)
+	if(!istype(M))
+		return
+	if(holder && holder.chem_temp >= fry_temperature)
+		boiling = TRUE
+	if(method in list(VAPOR, TOUCH)) //Directly coats the mob, and doesn't go into their bloodstream
+		if(boiling)
+			M.visible_message("<span class='warning'>The boiling oil sizzles as it covers [M]!</span>", \
+			"<span class='userdanger'>You're covered in boiling oil!</span>")
+			M.emote("scream")
+			playsound(M, 'sound/machines/fryer/deep_fryer_emerge.ogg', 25, TRUE)
+			var/oil_damage = (holder.chem_temp / fry_temperature) * 2 //Damage taken per unit
```

Comment:
This calculation's the problem with balance here. It scales way too sharply as temperature rises, meaning you can easily hit the damage cap with only 8 units (was 14 units with previous damage cap). Dividing instead of multiplying by two here would probably be optimal, as it'd increase the amount of units needed to hit the cap from 8 to ~32 with the cap of 35, or from 14 to 54 with the old cap.
The problem isn't that it's a 2-hit/3-hit crit, but rather that it's a very easily accessible 2-hit/3-hit crit.

Reply:
My worry about that is that cooking oil isn't very easy to get in bulk, and takes a decent amount of meat to get a respectable quantity. I'll have to think on it a bit.

Reply:
Taking a quick glimpse at the code, you can get 45 units of cooking oil out of monkeys with nothing but a knife, a food processor, and a grinder. Without a food processor, that'd be 10 units of cooking oil per monkey.
An assistant with a knife and a handful of monkey cubes really shouldn't be able to churn out one-time-use throwing stars with more damage than a fireaxe, but leaving it as an option for robust chefs and scientists would actually be pretty acceptable.

Reply:
Yeah, fair point!

Reply:
conversely fire axe has no prep.